doc/Programming-guide/Key-platform-features/Data-collection/File-system-log-appender/index.md

```diff
@@ -0,0 +1,234 @@
+
+Go to application's <b>Log appenders</b> configuration and add a new one.
+
+<img src="attach/data-collection-demo-application.png">
+
+Enter name of the new appender (we use “Sample FileSystem log appender”)
+
+Add Log metadata fields.
+
+Select _File_ appender type.
+
+Please, see [Creating file system log appender in Admin UI](#creating-file-system-log-appender-in-admin-ui) section for details.
+
+Verify that newly created appender has appeared in list.
+
+<img src="attach/verify-created-appender.png">
+
+Now run Data collection demo application:
+
+```bash
+java -jar DataCollectionDemo.jar
+2016-06-17 11:59:24,004 [main] INFO  o.k.k.d.d.DataCollectionDemo - Data collection demo started
+2016-06-17 11:59:25,457 [pool-2-thread-1] INFO  o.k.k.d.d.DataCollectionDemo - Kaa client started
+2016-06-17 11:59:25,459 [main] INFO  o.k.k.d.d.DataCollectionDemo - Log record {"level": "KAA_INFO", "tag": "TAG", "message": "MESSAGE_0", "timeStamp": 1466153965458} sent
+2016-06-17 11:59:25,460 [main] INFO  o.k.k.d.d.DataCollectionDemo - Log record {"level": "KAA_INFO", "tag": "TAG", "message": "MESSAGE_1", "timeStamp": 1466153965458} sent
+2016-06-17 11:59:25,460 [main] INFO  o.k.k.d.d.DataCollectionDemo - Log record {"level": "KAA_INFO", "tag": "TAG", "message": "MESSAGE_2", "timeStamp": 1466153965458} sent
+2016-06-17 11:59:25,460 [main] INFO  o.k.k.d.d.DataCollectionDemo - Log record {"level": "KAA_INFO", "tag": "TAG", "message": "MESSAGE_3", "timeStamp": 1466153965458} sent
+2016-06-17 11:59:25,460 [main] INFO  o.k.k.d.d.DataCollectionDemo - Log record {"level": "KAA_INFO", "tag": "TAG", "message": "MESSAGE_4", "timeStamp": 1466153965458} sent
+2016-06-17 11:59:27,102 [main] INFO  o.k.k.d.d.DataCollectionDemo - Received log record delivery info. Bucket Id [0]. Record delivery time [1644 ms].
+2016-06-17 11:59:27,102 [main] INFO  o.k.k.d.d.DataCollectionDemo - Received log record delivery info. Bucket Id [0]. Record delivery time [1644 ms].
+2016-06-17 11:59:27,102 [main] INFO  o.k.k.d.d.DataCollectionDemo - Received log record delivery info. Bucket Id [0]. Record delivery time [1644 ms].
+2016-06-17 11:59:27,102 [main] INFO  o.k.k.d.d.DataCollectionDemo - Received log record delivery info. Bucket Id [0]. Record delivery time [1644 ms].
+2016-06-17 11:59:27,102 [main] INFO  o.k.k.d.d.DataCollectionDemo - Received log record delivery info. Bucket Id [0]. Record delivery time [1644 ms].
+2016-06-17 11:59:27,103 [pool-2-thread-1] INFO  o.k.k.d.d.DataCollectionDemo - Kaa client stopped
+2016-06-17 11:59:27,104 [main] INFO  o.k.k.d.d.DataCollectionDemo - Data collection demo stopped
+```
+
+Let's verify that our logs have been persisted in the local file system. Go to Sandbox VM and open the file
+```/kaa_log_uploads/tenant_'number_of_tenant'/application_'your_application_token'/application.log```.
+In this example path to file ```application.log``` is ```kaa_log_uploads/tenant_1/application_82635305199158071549/```.
```

Comment:
Yes, you are right, the final compiled version looks correct. The issue with formatting I found (which made me think this is a NOTE) is GH-related.